test/e2e/runner/wait.go

```diff
@@ -30,5 +30,5 @@ func WaitUntil(testnet *e2e.Testnet, height int64) error {
 // waitingTime estimates how long it should take for a node to reach the height.
 // More nodes in a network implies we may expect a slower network and may have to wait longer.
 func waitingTime(nodes int) time.Duration {
-	return time.Duration(20+(nodes*4)) * time.Second
+	return time.Minute + (time.Duration(nodes) * (15 * time.Second))
```

Comment:
lol surely this is getting to the point where if this times out there's for sure something wrong with the code

Reply:
Given that the total runtime of the test doesn't change much, its probably more that this timeout gets used in a couple of cases, and not in others so I don't think we're hitting this very often.

Reply:
Yeah, the total runtime doesn't concern me. I completely agree that this is hardly being hit, I'm just concerned that even after increasing this, it still gets occasionally hit.

Reply:
its my sense that it's only getting hit in the perturbation cases.

Reply:
I thought we used `waitForNode()` after perturbations